rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -1360,7 +1360,7 @@ func Qualify_ForeignExchange_VanillaOption: <"Qualifies a product as FX Plain Va
 		and (economicTerms -> payout -> optionPayout -> feature is absent
 			or economicTerms -> payout -> optionPayout -> feature -> averagingFeature only exists)
 
-func Qualify_RepurchaseAgreement:
+func Qualify_SecuritiesFinancing:  <"Qualifies a product as generic Securities Financing; eg Securities Lending or Repurchase Agreement.">
```

Comment:
* This function should be renamed "...Finance" rather than "...Financing", as per indication given on the issue. Expectations will need to be re-generated accordingly - again.
* The function `Qualify_SecurityLendingAgreement` should also be removed